web-app/src/Components/Login/index.js

```diff
@@ -1,24 +1,17 @@
 import React from 'react';
 import Text from '../UI/Text';
+import LoginForm from '../Login/Form';
 import ContainerLayout from '../UI/ContainerLayout';
 
 const Login = () => (
-  <ContainerLayout header="Login">
+  <ContainerLayout >
     <Text>
```

Comment:
We don't seem to use that <Text> component anymore? Can we remove the tag and the import